eZ/Publish/API/Repository/Values/Content/VersionDiff/FieldDiff.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Values\Content\VersionDiff;
+
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition;
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class FieldDiff extends ValueObject
```

Comment:
DX: How to check if field values are equal/not equal in compared versions?

Reply:
https://github.com/ezsystems/ezpublish-kernel/pull/2819/commits/288517eddecb975045e05c7f919da428fea6f820